etc/apparmor/firejail-local

```diff
@@ -24,4 +24,4 @@
 #owner @{HOME}/.local/share/mullvad-browser/** ix,
 
 # Uncomment to opt-in to apparmor for torbrowser-launcher
-#owner @{HOME}/.local/share/torbrowser/tbb/{i686,x86_64}/tor-browser_*/Browser/** ix,
+#owner @{HOME}/.local/share/torbrowser/tbb/{i686,x86_64}/tor-browser/Browser/** ix,
```

Comment:
```suggestion
#owner @{HOME}/.local/share/torbrowser/tbb/{i686,x86_64}/tor-browser*/Browser/** ix,
```

This should make it work in both formats (new/old).

Reply:
I thought about adding it like that too. Here's my reasoning for not adding 'backward-compatibility' in this case. All software needs regular updating, that's proper secops 101. This goes especially for applications like tor-browser, on which users rely on for sensitive communications. A not-working TB > using an outdated versions.

Just my thoughts, I'm open to change it if you feel it's absolutely necessary.